### DISCUSSION TITLE
management port bonding

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#10026] [General]
hi,

is it possible to create bonding for the management port ?

**Comment by InfosecGoon** [comment]
It should be. Is this for fault tolerance / redundancy or for additional bandwidth?

**Reply by udi-mosh** [reply]
both, i think...
i want redundancy as everything should in a production environment and if the management port is the one receiving syslogs, then bandwidth should take into consideration although 1GB Ethernet port should be enough.
the setup does not offer an option to create such bond

**Reply by InfosecGoon** [reply]
You should be able to create a bond manually - make sure that you assign it the same IP address that you're currently using for your management port, and update the "mainint" directive in the minion file for the node to say "bond0" rather than the current interface name.

That said, if you're using a distributed architecture you could also run a Receiver Node to provide redundancy and failover in the event that your Manager has an issue and can no longer ingest.

**Reply by udi-mosh** [reply]
@InfosecGoon thank you very much.
since i am using so iso to install and upgrade, won't it be a problem ? the documentation indicates that nothing should be changed (besides conf files}.
